src/pwd.cpp

```diff
@@ -0,0 +1,9 @@
+#include "commands.h"
+
+#include <unistd.h>
+
+// pwd command (does not need to free malloc)
+void seashell::pwd(const std::string &) {
+    const auto * dir_name = get_current_dir_name();
+    puts(dir_name);
```

Comment:
```suggestion
    const auto * dir_name = get_current_dir_name();
    puts(dir_name);
    free(dir_name);
```
Forgot to free the string.